release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -105,6 +105,12 @@ To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://gi
 
 #### New Features
 
+- Added debugging support for package dependency sources. With this feature, now you can step-in and debug the sources
+  related to the following dependency package types, that are used within your ballerina program.
+  - language library sources
+  - standard library sources
+  - imported modules from the central
```

Comment:
Shall we make this a heading instead?

```suggestion
##### Debugging Support for Dependencies

With this feature you can now step-in and debug the sources of the following dependencies used within your Ballerina program.
  - language library
  - standard library
  - modules imported from Ballerina Central
```

Reply:
@MaryamZi +1.. shall we also make `language library` -> `language libraries` and `standard library` -> `standard libraries` in your suggestion?

Reply:
@praneesha WDYT?

Reply:
+1